src/applications/vaos/reducers/newAppointment.js

```diff
@@ -65,7 +66,7 @@ const initialState = {
   pageChangeInProgress: false,
   loadingSystems: false,
   loadingEligibility: false,
-  loadingFacilityDetails: false,
+  facilityDetailsStatus: FETCH_STATUS.notStarted,
```

Comment:
I thought you had added this status, but it looks like it's used in some other pages. You might need to update all those instances, or just leave it for now and we can refactor later.